api/lib/controllers/meeting.js

```diff
@@ -229,7 +229,27 @@ module.exports = {
   async index( req, res ) {
     const subject_email = req.credentials.user.email;
     const subject_id = req.credentials.sub;
-    const { limit = 0, skip = 0 } = req.query; // Thanks tom
+    const {  filters = `{ "owners": [], "name": ""}` } = req.query; // Thanks tom
```

Comment:
Express with automatically parse the params. The reason they are being returned like this is because the `filters` object on the frontend is an object within an object. It might be nice to validate the parameters though.